### DISCUSSION TITLE
CV-SDK  containerized on Axis Q1615 Mk III - ARTPEC-7

### DISCUSSION THREAD

**Discussion by swarali-nandgaonkar** [AxisCommunications/acap-computer-vision-sdk-examples#198] [Q&A] [answered]
As the Q1615 MKIII work with ARTPEC-7 i.e. I assume the architecture is arm7l i.e. arm7hf. The OS is 11.11.73. For this specific case, are are no examples in the CV examples library. I specifically need help for the dockerfile. Where can I find help/examples related to this?

**Comment by vivekatoffice** [comment, marked as answer]
Hi @swarali-nandgaonkar ,
You can still use the v1.15 tag for armv7hf: 

![image](https://github.com/user-attachments/assets/dba16025-b396-46f1-8da0-cfcc8cc8e54a)

**Comment by swarali-nandgaonkar** [comment]
Hi @vivekatoffice, thank you for your response. I tried building the hello-world example from this repo, and see an error while building the image- 

![image](https://github.com/user-attachments/assets/2a364b86-92d7-4f94-9136-7a310e0633ed)

I'm not sure if this has something to do with docker or acap or my personal settings. I was also unable to build any of my previously working applications because of this error. Could you please help if you have an idea about this?

**Reply by swarali-nandgaonkar** [reply]
Update: 

Modifying dockerfile with 
`ARG DEVICE_PLATFORM=linux/arm/v7`
`FROM --platform=$DEVICE_PLATFORM ubuntu:${UBUNTU_VERSION} AS runtime-image-armv7hf`

instead of 

`FROM arm32v7/ubuntu:${UBUNTU_VERSION} as runtime-image-armv7hf`
`FROM arm64v8/ubuntu:${UBUNTU_VERSION} as runtime-image-aarch64`

and installing qemu to allow build for a different architecture solved the problem with building the image. 
`docker run --rm --privileged multiarch/qemu-user-static --credential yes --persistent yes`

Could anyone explain why this is needed at all? It wasn't needed earlier.
